pkg/handlers/generic/lifecycle/csi/nutanix-csi/handler.go

```diff
@@ -151,12 +151,17 @@ func (n *NutanixCSI) handleHelmAddonApply(
 		)
 	}
 
-	helmChart, err := n.helmChartInfoGetter.For(ctx, log, config.NutanixStorageCSI)
+	storageChart, err := n.helmChartInfoGetter.For(ctx, log, config.NutanixStorageCSI)
 	if err != nil {
-		return fmt.Errorf("failed to get values for nutanix-csi-config %w", err)
+		return fmt.Errorf("failed to get helm chart %q: %w", config.NutanixStorageCSI, err)
 	}
 
-	hcp := &caaphv1.HelmChartProxy{
+	snapshotHelmChart, err := n.helmChartInfoGetter.For(ctx, log, config.NutanixSnapshotCSI)
```

Comment:
nit: similar to `storageChart`, `snapshotChartProxy` and `storageChartProxy`
```suggestion
	snapshotChart, err := n.helmChartInfoGetter.For(ctx, log, config.NutanixSnapshotCSI)
```

Reply:
Applied and pushed to PR branch.